docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -555,6 +555,35 @@
           }
         }
       }
+    },
+    "generated": {
```

Comment:
This always going to be generated? Can your extractor find everything or could it be required for an author to manually edit this?

Reply:
Yeah, I imagined data under this property to always be automatically generated. (It can still be edited manually as well in case there is some kind of failure with the extraction, but ideally I think it's not meant to.)

Reply:
Okay 😅